src/containers/MintWithTicket/Panel/PanelContext.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from "./PanelContext.module.scss"
+import { PanelGroup } from "./PanelGroup"
+import { Switch } from "components/Input/Switch"
+
+export interface PanelContextProps {
+  context: "minting" | "standalone"
+  onChangeContext: (c: "minting" | "standalone") => void
+}
+
+export function PanelContext({ context, onChangeContext }: PanelContextProps) {
+  return (
+    <PanelGroup title="Context" description="Useful description...">
```

Comment:
any suggestions for description here? @maerzhase

Reply:
i'd suggest something like: "Toggle the execution context of the artwork between "minting" and "standalone""